iac-extensions/terraform/src/test/resources/checks/AuditLogMemberExclusionCheck/gcp_audit_config.tf

```diff
@@ -0,0 +1,25 @@
+resource "google_project_iam_audit_config" "noncompliant" {
+  audit_log_config {
+    exempted_members = [ # Noncompliant {{Make sure excluding members activity from audit logs is safe here.}}
+  # ^^^^^^^^^^^^^^^^
+      "user:eric.therond@sonarsource.com",
+    ]
+  }
+}
+
+resource "google_project_iam_audit_config" "compliant" {
+  audit_log_config {
+    exempted_members = [] # Compliant empty list
+  }
+
+  audit_log_config { # Compliant missing equals empty
+  }
+}
+
+resource "non_google_project_iam_audit_config" "coverage" {
+  audit_log_config {
+    exempted_members = [
+      "user:eric.therond@sonarsource.com",
+    ]
+  }
+}
```

Comment:
I would add a test where `exempted_members` is not a list literal, e.g.;
```
exempted_members = var.something
```
And maybe a case where `exempted_members` is omitted.